common/settings.cc

```diff
@@ -8,6 +8,10 @@
 
 namespace blink {
 
+Settings::Settings() = default;
```

Comment:
We usually add an empty line between method methods. Here and elsewhere in this commit.

Reply:
Fixes. I was expecting that clang-format would add those spaces for me. I guess it's not that smart...